src/repm-invoker.ts

```diff
@@ -66,10 +66,9 @@ export class REPMWasmInvoker {
 }
 
 type OpenRequest = {
-  // eslint-disable-next-line @typescript-eslint/naming-convention
-  use_memstore: boolean;
+  useMemstore: boolean;
```

Comment:
This was not an issue before because our type testing of invoke needs more work. With this change it became an issue because we read the response.